frontend/server/src/main/java/org/pytorch/serve/wlm/ModelManager.java

```diff
@@ -211,15 +213,36 @@ private void setupModelDependencies(Model model)
                             + requirementsFilePath; // NOPMD
 
             String[] envp =
-                    EnvironmentUtils.getEnvString(configManager.getModelServerHome(), null, null);
+                    EnvironmentUtils.getEnvString(
+                            model.getModelDir().getAbsolutePath(), null, null);
+
             Process process =
                     Runtime.getRuntime()
                             .exec(
                                     packageInstallCommand,
                                     envp,
                                     model.getModelDir().getAbsoluteFile());
+
             int exitCode = process.waitFor();
+
             if (exitCode != 0) {
+
+                String line;
+                // process's stdout is InputStream for caller process
+                logger.error("Dependency installation stdout:");
+                BufferedReader brdr =
+                        new BufferedReader(new InputStreamReader(process.getInputStream()));
+                while ((line = brdr.readLine()) != null) {
+                    System.out.println(line);
+                }
+
+                // process's stderr is ErrorStream for caller process
+                logger.info("Dependency installation stderr:");
+                brdr = new BufferedReader(new InputStreamReader(process.getErrorStream()));
+                while ((line = brdr.readLine()) != null) {
+                    System.out.println(line);
```

Comment:
same here.

Reply:
I've added the stdout and stderr to the logger.